lesson_6_hw.js

```diff
@@ -0,0 +1,54 @@
+/*
+1. Написати функцію, яка перетворює строку на масив слів,
+    сортує його за алфавітним порядком і повертає (у масиві не повинно бути пробілів як елементів)
+ */
+
+const str = "Це довільний текст для створення масиву слів ";
+function stringToArray(str) {
+    const transformToArr = str.split(" ").filter(word => word !== '')
+    const sortArr = transformToArr.map(word => word.toLowerCase()).sort();
+    return sortArr.sort()
+}
+
+console.log(stringToArray(str));    // виводить [ 'для', 'довільний', 'масиву', 'слів', 'створення', 'текст', 'це' ]
+
+/*
+2. Написати функцію, яка видаляє з масива елементи, що дублюються, та повертає масив оригінальних елементів,
+відсортованих за зростанням
+ */
+const initialArray = [1, 4, 6, 6, 7, 5, 34, 5, 1000, 66, 1];
+function removeDuplicatesAndSort(arr) {
+    uniqueSet = new Set(arr);
+    uniqueArr = Array.from(uniqueSet);
+    uniqueArr.sort((a, b) => a - b);
+    return uniqueArr.join(", "); 

```

Comment:
```suggestion
    return uniqueArr
```
має повернутися масив